src/garage/envs/pybullet/__init__.py

```diff
@@ -0,0 +1,13 @@
+"""Wrappers for the Pybullet based gym environments.
+
+See https://github.com/bulletphysics/bullet3/tree/master/examples/pybullet
+"""
+try:
+    import pybullet_envs  # noqa: F401
+except Exception as e:
+    raise ImportError("To use garage's pybullet wrappers, please install "
+                      'garage[pybullet].')
+
+from garage.envs.pybullet.pybullet_env import PybulletEnv
```

Comment:
pybullet styles themselves as PyBullet, so this should either be `PyBulletEnv`, or simply `BulletEnv` is probably sufficient (since PyBullet is just the Python interface for Bullet).